Jenkinsfile

```diff
@@ -319,13 +318,31 @@ def job = {
                     sh "docker push ${config.dockerRegistry}${dockerRepo}:master-latest"
 
                     if (config.release) {
-                        sh "docker tag ${config.dockerRegistry}${dockerRepo}:${config.docker_tag} ${config.dockerRegistry}${dockerRepo}:${config.docker_tag}-${env.BUILD_NUMBER}"
-                        sh "docker push ${config.dockerRegistry}${dockerRepo}:${config.docker_tag}-${env.BUILD_NUMBER}"
+                        // docker_tag for release builds does not include the build number
+                        // here we add it back so an image version with the build number always exists
+                        sh "docker tag ${config.dockerRegistry}${dockerRepo}:${config.docker_tag} ${config.dockerRegistry}${dockerRepo}:${config.docker_tag}-rc${env.BUILD_NUMBER}"
+                        sh "docker push ${config.dockerRegistry}${dockerRepo}:${config.docker_tag}-rc${env.BUILD_NUMBER}"
                     }
                 }
             }
         }
     }
+
+    // Build CCloud ksqlDB image
+
+    if (!config.release && !config.isPrJob) {
+        stage('Trigger CCloud ksqlDB Docker image job') {
+            build job: "confluentinc/cc-docker-ksql/master",
+                wait: false,
+                parameters: [
+                    booleanParam(name: "NIGHTLY_BUILD", value: true),
+                    string(name: "NIGHTLY_BUILD_NUMBER", value: "${env.BUILD_NUMBER}"),
```

Comment:
1 nit here: would it be better to just pass in an opaque version string here rather than the beta version and the nightly build number and having the cc-docker-ksql build reconstruct the ksqldb rc version? It would be easier to keep it consistent this way (in case we change the format of the ksqldb rc version).

Reply:
Discussed offline. Decision was to remove the parameter `NIGHTLY_BUILD_NUMBER` and have the cc-docker-ksql build construct its docker image tag from `KSQLDB_ARTIFACT_VERSION` directly. The beta version params are separate.

Reply:
Nvm, decided to leave this as is in order to allow more flexibility for the downstream image tag format.